src/i18n/en-US/systemWorkspace.ts

```diff
@@ -0,0 +1,73 @@
+export type HelpLinkType = {
+  header: string;
+  linkText: string;
+  link: string;
+  external?: boolean;
```

Comment:
nit: i think it might be more clear to make this not-nullable and then explicitly mark `external: false` below where necessary

Reply:
This can be ok. It's typical to assume links in the app are generally internal. Defaulting to false isn't a big surprise.